dev-vcs/tig/tig-2.2.recipe

```diff
@@ -4,13 +4,13 @@ Tig functions mainly as a git repository browser. It can also help in the \
 staging changes for commit at chunk level and act as a pager for output \
 different git commands."
 HOMEPAGE="http://jonas.nitro.dk/tig/"
-SOURCE_URI="http://jonas.nitro.dk/tig/releases/tig-2.1.tar.gz"
-CHECKSUM_SHA256="306287f684f57563a53abf1cf46149e0d30c6b500fbc0c39e9bc059506373cb0"
-REVISION="1"
+COPYRIGHT="2006-2016 Jonas Fonseca"
 LICENSE="GNU GPL v2"
-COPYRIGHT="2006-2014 Jonas Fonseca"
+REVISION="1"
+SOURCE_URI="http://jonas.nitro.dk/tig/releases/tig-$portVersion.tar.gz"
```

Comment:
I wanted to suggest a few extra changes in a comment review, but GitHub does not yet allow to add comments to lines that have not changed... So here's a single comment with some changes that you might wish to apply:
* Add a blank line after **`SECONDARY_ARCHITECTURES="x86"`** and this block:
```
if [ "$targetArchitecture" != x86_gcc2 ]; then
	commandBinDir=$binDir
else
	commandBinDir=$prefix/bin
fi
```

* then, in PROVIDES, replace
```
	cmd:tig$secondaryArchSuffix = $portVersion
```
by
```
	cmd:tig = $portVersion
```

* and finally, in INSTALL, replace
```
	runConfigure ./configure
```
by
```
	runConfigure  --omit-dirs binDir ./configure --bindir=$commandBinDir
```

These changes will allow tig to be called without the **`-x86`** suffix on x86_gcc2.
(That's what we have with lcms-2.8, see [media-libs/lcms/](https://github.com/haikuports/haikuports/tree/master/media-libs/lcms).)

Of course, all this only makes sense if you are sure that the recipe does not build on x86_gcc2 primary arch. In case a trivial patch is enough to make it build on x86_gcc2 primary arch then it might be worth adding it and dropping support for secondary arch build. So if anyone want's to dig that question, it would be cool!